src/components/Import/Import.jsx

```diff
@@ -1,14 +1,10 @@
 import React from "react";
 import { FileUploader } from "carbon-components-react";
-import styles from "./from_stix.module.scss";
-import { loadJsonFromDisk } from "./utils";
+import styles from "./import.module.scss";
 import { useDispatch } from "react-redux";
-import {
-  updateMappingsFromFile,
-  clearMappings,
-} from "../../store/actions/from_stix";
 
-const Import = () => {
+const Import = (props) => {
```

Comment:
please add proptypes for type checking https://reactjs.org/docs/typechecking-with-proptypes.html